### DISCUSSION TITLE
Welcome to Serenity Discussions!

### DISCUSSION THREAD

**Discussion by VictorTomaili** [serenity-is/Serenity#5299] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dfaruque** [comment]
Thanks for this. Hope we will build a more strong community in the future.

I am a serenity community member since 2015.
I have found this gem (Serenity) by searching for a rapid-development platform and still, I have not found another one that can compete with serenity. 

Thank you a lot @volkanceylan, @VictorTomaili, and other members for this dream platform.

**Comment by JohnRanger** [comment]
Thanks @VictorTomaili for bringing this into action.

I have found Serenity around 2015/2016 and I am sticking here around since then. As @dfaruque stated here, I have also not found anything close to the ease and speed of developing web-based business Apps. It is even possible to produce nice, user-friendly apps with it for a non-professional dev like me ;-)

Thanks also from my side to @volkanceylan and @VictorTomaili  for this superb platform.

With kind regards,

John
